src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,140 +1,123 @@
+type Plugins = NonNullable<React.ComponentProps<typeof Tippy>['plugins']>;
+type Plugin = Plugins[number];
 
-  const handleOnClickOutside = (event: MouseEvent | TouchEvent) => {
-    if (
-      ref.current &&
-      !ref.current.contains(event.target as HTMLElement) &&
-      buttonRef.current &&
-      !buttonRef.current.contains(event.target as HTMLElement)
-    ) {
-      setIsActive(false);
-      setTimeout(() => {
-        if (isActive) {
-          buttonRef.current.focus();
+/**
+ * ```ts
+ * import {Tooltip} from "@chanzuckerberg/eds-components";
+ * ```
+ *
+ * A styled tooltip built on Tippy.js.
+ *
+ * https://atomiks.github.io/tippyjs/
+ * https://github.com/atomiks/tippyjs-react
+ */
+export const Tooltip = ({
+  variant = 'light',
+  align = 'top',
+  className,
+  text,
+  ...rest
+}: TooltipProps) => {
+  // Hides tooltip when escape key is pressed, following:
+  // https://atomiks.github.io/tippyjs/v6/plugins/#hideonesc
+  const hideOnEsc: Plugin = {
+    name: 'hideOnEsc',
+    defaultValue: true,
+    fn: ({ hide }) => {
+      function onKeyDown(event: KeyboardEvent) {
+        if (event.key === 'Escape') {
+          hide();
         }
-      }, 1);
-    }
+      }
+      return {
+        onShow() {
+          document.addEventListener('keydown', onKeyDown);
+        },
+        onHide() {
+          document.removeEventListener('keydown', onKeyDown);
+        },
+      };
+    },
   };
-
-  /**
-   * Handle Keydown
-   * 1) Remove isActive state and focus on the tooltip button
-   */
-  function onKeyDown(e: any) {
-    if (e.code === ESCAPE_KEYCODE || e.code === TAB_KEYCODE) {
-      setIsActive(false);
-      setTimeout(() => {
-        if (isActive) {
-          buttonRef.current.focus();
-        }
-      }, 1);
-    }
-  }
-  const componentClassName = clsx(styles['tooltip'], className, {
-    [styles['eds-is-active']]: isActive,
-    [styles['tooltip--right']]: align === 'right',
-    [styles['tooltip--below']]: align === 'below',
-    [styles['tooltip--left']]: align === 'left',
-  });
-
   return (
-    <div className={componentClassName} {...other}>
-      <span className={styles['tooltip__trigger']}>
-        <Button
-          variant="bare"
-          hideText
-          iconPosition="after"
-          iconName="question-mark-circle"
-          aria-label={buttonText}
-          text={buttonText}
-          type="button"
-          onClick={onClick}
-          buttonRef={buttonRef}
-        />
-      </span>
-      <span
-        ref={ref}
-        className={styles['tooltip__content']}
-        aria-hidden={!isActive}
-        tabIndex={isActive ? 0 : -1}
-        onKeyDown={onKeyDown}
-      >
-        {children}
-      </span>
-    </div>
+    <Tippy
+      {...rest}
+      className={clsx(
+        styles.tooltip,
+        className,
+        variant === 'light' && styles['tooltip--light'],
+        variant === 'dark' && styles['tooltip--dark'],
```

Comment:
Do you know if we've cleared with BM if we're using ampersands or object literals for these subjective classes? The code guidelines still has it using the object literals, but I think we want to keep using the ampersands. https://github.com/chanzuckerberg/edu-design-system/blob/next/docs/CODE_GUIDELINES.md#define-componentclassname

Reply:
Oooh good q, I do like ampersands b/c I feel like it's easier to read

Reply:
I think they wanted to see an example in code first. So here it is! @bradfrost @ifrost1 @csnizik any concerns with this syntax?

Reply:
Via offline eng sync, committing to this style of syntax